src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -773,6 +773,22 @@ protected theorem extractLsb_ofNat (x n : Nat) (hi lo : Nat) :
 @[simp] theorem extractLsb_toNat (hi lo : Nat) (x : BitVec n) :
   (extractLsb hi lo x).toNat = (x.toNat >>> lo) % 2^(hi-lo+1) := rfl
 
+@[simp] theorem extractLsb'_toInt (s m : Nat) (x : BitVec n) :
```

Comment:
```suggestion
@[simp] theorem extractLsb'_toInt {s m : Nat} {x : BitVec n} :
```

Since the variable `s`, `m`, `x` occur on the LHS and the RHS, we should standardize and make these implicit.